starfish/core/codebook/codebook.py

```diff
@@ -552,6 +552,13 @@ def decode_metric(
 
         self._validate_decode_intensity_input_matches_codebook_shape(intensities)
 
+        # add empty metadata fields and return
+        if intensities.sizes[Features.AXIS] == 0:
```

Comment:
what does the code below do (prior to your change)? I assume it didn't create a zero-length array since you're doing this, but what did it create?

Reply:
@ambrosejcarr